diff --git a/node_modules/@formulajs/formulajs/lib/esm/index.mjs b/node_modules/@formulajs/formulajs/lib/esm/index.mjs
index ed08e63..fe1ef26 100644
--- a/node_modules/@formulajs/formulajs/lib/esm/index.mjs
+++ b/node_modules/@formulajs/formulajs/lib/esm/index.mjs
@@ -1450,7 +1450,7 @@ function TEXTJOIN(delimiter, ignore_empty, ...args) {
     return na
   }
 
-  delimiter = delimiter ?? '';
+  delimiter = delimiter || '';
 
   let flatArgs = flatten(args);
   let textToJoin = ignore_empty ? flatArgs.filter((text) => text) : flatArgs;
